parse/IntValueRefParser.cpp

```diff
@@ -33,7 +33,6 @@ parse::detail::simple_int_parser_rules::simple_int_parser_rules(const parse::lex
         |   tok.FleetID_
         |   tok.PlanetID_
         |   tok.SystemID_
-        |   tok.ContainerID_
```

Comment:
What is your definition of "unused"?

https://github.com/freeorion/freeorion/blob/master/universe/ValueRefs.cpp#L1051

Reply:
Not mentioned in FOCS files. Some of them even not used in parser code.

Reply:
The ones that aren't even reference in parser code are fine to remove.